Sources/PackageGraph/CompoundPackageContainerProvider.swift

```diff
@@ -0,0 +1,81 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import Dispatch
+
+import PackageLoading
+import PackageModel
+import PackageRegistry
+import SourceControl
+
+import TSCBasic
+import TSCUtility
+
+public class CompoundPackageContainerProvider: PackageContainerProvider {
```

Comment:
total naming bike shedding and not a strong opinion, but I prefer `Composite ` over `Compound`

Reply:
Agreed. Renamed in 9c061c8b.